tests/system/grant_test.go

```diff
@@ -0,0 +1,115 @@
+//go:build system_test
+
+package system
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
+)
+
+func TestGrantStoreCodePermissionedChain(t *testing.T) {
+	sut.ResetChain(t)
+	cli := NewWasmdCLI(t, sut, verbose)
+
+	// add genesis account with some tokens
+	account1Addr := cli.AddKey("account1")
```

Comment:
account1 and 2 are not very verbose. Can think about the role that the address represents?
Maybe something like chain_authorization and dev ?